.tool-versions

```diff
@@ -1,2 +1,3 @@
 # Also update ci.yml
-erlang 27.1.3
+erlang 27.2.3
```

Comment:
We should stick with the current `erlang 27.1.3` version that we run in production.

Reply:
Yes, will fix that 👌🏽

Reply:
@weppos we will be doing a release and there is also a tag with `27.1.3` we can use. We are not running a risk updating the version to `27.2.4`